packages/stream_video_flutter/lib/src/call_participants/call_participant.dart

```diff
@@ -202,7 +202,7 @@ class StreamCallParticipant extends StatelessWidget {
                   ),
                   if (participant.reaction != null)
                     Align(
-                      alignment: Alignment.topRight,
+                      alignment: Alignment.topCenter,
```

Comment:
Since this is in our SDK shouldn't we have it configurable? 🤔

Reply:
Yes true, I would leave it for now and addressed it in separate task because I think we need to redo the reaction displaying component. There should be a builder callback for it so users can customize it and we should make it disapear after some time.